nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
+
+4.   Shortly after selecting the appropriate boot drive, you should be presented
+with a menu with different Installer options. Leave the default and wait
+(or press <kbd>Enter</kbd> to speed up).
+
+5.   The graphical images will start the desktop environment and the graphical
+installer, which can take some time. The minimal images will just have
+a command line.
+
+## Graphical Installation {#sec-installation-graphical}
+
+The graphical installer is recommended for desktop users and will guide you
+through the installation.
+
+1.   In the "Welcome" screen, you can select the language of the Installer and
+     the installed system.
+
+     ::: {.tip}
+     Leaving the language as "American English" will help to be able to copy
+     and paste error messages in a search engine or issue report.
+     :::
+
+2.   Next you should choose your location to have the timezone set correctly.
+     You can actually click on the map!
+
+     ::: {.note}
+     The installer will use an online service to guess your location based on
+     your public IP address.
+     :::
+
+3.   Then you can select the keyboard layout. The default keyboard model should
+     work well with most desktop keyboards. If you have a special keyboard or
+     notebook, your model might be in the list. Select the language you are most
+     comfortable typing in.
+
+     ::: {.tip}
+     There are special keyboard layouts that are optimized to be ergonomic like
+     [NEO](https://www.neo-layout.org/),
+     [Dvorak](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout) and
+     [Colemak](https://colemak.com/), in opposition to the classic QWERTY layout
+     which comes from typewriters and has not changed much since 1873.
+     You might want to check them out.
+     :::
 
-The installation media can be burned to a CD, or now more commonly,
-"burned" to a USB drive (see [](#sec-booting-from-usb)).
+4.   On the "Users" screen, you have to type in your name, login name and
+     password. You can also enable an option to automatically login to the
+     desktop.
 
-The installation media contains a basic NixOS installation. When it's
-finished booting, it should have detected most of your hardware.
+5.   Then you have the option to choose a desktop environment. If you want to
+     create a custom setup with a window manager, you can select "No desktop".
 
-The NixOS manual is available by running `nixos-help`.
+     ::: {.tip}
+     If you don't have a favorite desktop and don't know which to choose, you
+     can stick to either GNOME or Plasma. They are quiet different from their
+     design, so you should be able to choose which one you like better.
+     They are both popular choices and well tested on NixOS.
+     :::
+
+6.   You have the option to allow unfree software in the next screen.
+
+7.   The easiest setting in the "Partitioning" screen is "Erase disk", which will
+     delete all data from the selected disk and installs the system on it.
```

Comment:
Either

```suggestion
     delete all data from the selected disk, and installs the system on it.
```

or

```suggestion
     delete all data from the selected disk and install the system on it.
```

It depends though on how the sentence is split I guess? I assume the difference is between "delete and install" or "delete, then installs".